### PR TITLE
Updated CDISC format

### PR DESCRIPTION
**Pull request by gaaliciA1990**
Added new table to CDISC subsets based on excel provided by Erin. 

1. Created new CDISC table in subset-details.component.html
a. Added new columns
b. populated data in the columns. 
c. updated "other" format to exclude CDISC
3. Added check for CDISC in subset-details.component.ts
4. This includes the updates for Java 17 and ena links as well, given this is based on my updated branch for Java17

### CODE REVIEW

File: web/frontend/src/app/component/subset-details/subset-details.component.ts [hunk 1]
```diff
@@ -163,7 +169,7 @@ export class SubsetDetailsComponent implements OnInit {
         if (!(synonymSourceMap.has(synonym['source']))) {
           synonymSourceMap.set(synonym['source'], synonym['name']);
         }
-        else {
+        else {5
```
Comment (peter-va) [inline web/frontend/src/app/component/subset-details/subset-details.component.ts:172]: just a little typo there

**Review by peter-va — APPROVED** [review]
Looks good to me

**Comment by gaaliciA1990** [comment]
Pending testing on the dev environment with the created tag. Jenkins still pending java 17

**Comment by gaaliciA1990** [comment]
rebased from dev branch

**Comment by gaaliciA1990** [comment]
tag built and deployed for this branch. Confirmed changes are live and working as expected.

**Review by bcarlsenca — APPROVED** [review]
merging

**Pull request merged**